### PR TITLE
Julia summary-based guide to get you up & running

### PR DESCRIPTION
**Pull request by dimiboeckaerts**

### CODE REVIEW

File: chapters/00.Introduction/00-GetGoingWithJulia.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Let's get going with Julia! This summary-based guide into programming with Julia is meant for students and researchers with a general background in programming but no specific knowledge of Julia. The first two sections get you up and running in Julia, while the third and fourth section dive into some more (bio)engineering-oriented material. Finally, the last section provides additional resources to explore more from Julia. Happy learning!
+
+[TOC]
+
+
+## The basics
+
+What is Julia again?
+https://julialang.org/learning/getting-started/
```
Comment (MichielStock) [inline chapters/00.Introduction/00-GetGoingWithJulia.md]: Maybe tidier if you put this as links in the text?
[What is Julia again?](https://julialang.org/learning/getting-started/) etc?

File: chapters/00.Introduction/00-GetGoingWithJulia.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Why should you learn Julia?
+https://blog.goodaudience.com/10-reasons-why-you-should-learn-julia-d786ac29c6ca
```
Comment (MichielStock) [inline chapters/00.Introduction/00-GetGoingWithJulia.md]: ditto

File: chapters/00.Introduction/00-GetGoingWithJulia.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+Let's download & install Julia. You can either download the language itself and engage with it through the command-line (or notebooks, see below):
```
Comment (MichielStock) [inline chapters/00.Introduction/00-GetGoingWithJulia.md]: Don't know this. I think Visual Studio Code is also used a lot. Best mention this as well.

File: chapters/00.Introduction/00-GetGoingWithJulia.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+https://juliacomputing.com/products/juliapro.html
+
+First baby steps around the block:
+https://docs.julialang.org/en/v1/manual/getting-started/
+
+Learn the basics of Julia:
+https://juliabyexample.helpmanual.io/
+https://learnxinyminutes.com/docs/julia/
+
+Learn more about types, multiple dispatch and structs:
```
Comment (MichielStock) [inline chapters/00.Introduction/00-GetGoingWithJulia.md]: add comment: "Not needed to understand into detail for STMO"

File: chapters/00.Introduction/00-GetGoingWithJulia.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+
+## Notebooks & Visualization
+
+### Jupyter notebook & IJulia
+Coupling Julia to Jupyter notebooks is done with IJulia:
+https://www.youtube.com/watch?v=oyx8M1yoboY
+
+### Pluto
+Pluto is a lightweight, reactive notebook for Julia, with a lot of useful features:
```
Comment (MichielStock) [inline chapters/00.Introduction/00-GetGoingWithJulia.md:36]: Pluto will be used for the projects while we use Jupyter notebook for the exercises.

File: chapters/00.Introduction/00-GetGoingWithJulia.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+## Where to get additional help?
+
+A Julia (vs Python vs MATLAB) cheatsheet! https://cheatsheets.quantecon.org/
+
+Free courses at Julia Academy: https://juliaacademy.com/courses
+
+Community: https://julialang.org/community/
+
+Books: https://julialang.org/learning/books/
+
```
Comment (MichielStock) [inline chapters/00.Introduction/00-GetGoingWithJulia.md:150]: Link naar de gitter voor de cursus.

**Review by MichielStock — CHANGES_REQUESTED** [review]
Good job! Please fix the tiny suggestions.

**Comment by MichielStock** [comment]
In addition, can you link this to the readme.md file?

**Comment by dimiboeckaerts** [comment]
Check!

**Pull request merged**